package.json

```diff
@@ -5,6 +5,7 @@
   "main": "src/index.js",
   "scripts": {
     "lint": "prettier --check \"functions/src/**/*.{js,jsx,ts,tsx}\"",
+    "format": "prettier --write \"functions/src/**/*.{js,jsx,ts,tsx}\"",
```

Comment:
This should generally be integrated in your editor. While I have nothing against adding this, you shouldn't need to run this manually.